api-2.2/src/main/java/org/openmrs/module/reporting/cohort/definition2_2/ConditionCohortDefinition.java

```diff
@@ -0,0 +1,71 @@
+
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.reporting.cohort.definition2_2;
```

Comment:
Why do you have this package?  Is the 2_2 suffix necessary?

Reply:
removed the 2_2 suffix